react-typescript/tmpl/src/index.ts

```diff
@@ -4,7 +4,7 @@ import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-insta
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
-let mainWindow;
+let mainWindow: Electron.BrowserWindow | null = null;
```

Comment:
I'm not too sure how typescript typings are resolved, but setting the type to `BrowserWindow` resulted in an error, while `Electron.BrowserWindow` doesn't.  You can quickly test it out with `tsc --noEmit`.